.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+
+# We use a docker image as the basis for our build, so that all the toolchains we use
+# are already installed and the build can start running right away. It also guarantees
+# the environment is portable and reproducible on your local machine.
+var_1: &docker_image angular/ngcontainer:0.3.0
+
+# CircleCI lets us pick the key for storing one or more caches, to speed up subsequent builds.
+# We can use this to avoid re-fetching our dependencies from npm on every build.
+# To ensure we don't load a stale cache, we invalidate it based on the entries in the key:
+# - the checksum of Yarn's lock file
+# - the branch we are on, which really shouldn't be needed since the yarn lock file should be hermetic
+# - the docker image tag, working around an issue we saw where changing docker images causes permission
+#   errors when restoring the cache, like when the user we run as changes
+var_2: &cache_key angular-{{ .Branch }}-{{ checksum "yarn.lock" }}-0.3.0
+
+# Each job will inherit these defaults
+anchor_1: &job_defaults
+  working_directory: ~/ng
+  docker:
+    - image: *docker_image
+
+# After checkout, rebase on top of master, because we want to test the proposed merge of a
+# onto the target branch, not just test what's on the user's fork.
+# Similar to travis behavior, but not quite the same.
+# See https://discuss.circleci.com/t/1662
+anchor_2: &post_checkout
+  post: git pull --ff-only origin "refs/pull/${CI_PULL_REQUEST//*pull\//}/merge"
+
+
+# These jobs will run in parallel, and report separate statuses to GitHub PRs
+jobs:
+  lint:
+    <<: *job_defaults
+    steps:
+      - checkout:
+          <<: *post_checkout
+      - run: './pre-commit.sh'
+      - run: 'buildifier -mode=check $(find . -type f \( -name BUILD.bazel -or -name BUILD \)) ||
```

Comment:
Does `yarn buildifier` run the same buildifier as this `buildifier` in the docker image?

Reply:
@oferb not really, `buildifier` comes bundled with docker image but in `yarn buildifier` we download it. so it may be a good idea to replace CI command with `bazel run //tools:buildifier`

Reply:
@oferb not really, `buildifier` comes bundled with docker image but in `yarn buildifier` we download it. so it may be a good idea to replace CI command with `bazel run //tools:buildifier`